.mergify.yml

```diff
@@ -193,6 +193,19 @@ pull_request_rules:
         labels:
           - "backport"
         title: "[{{ destination_branch }}](backport #{{ number }}) {{ title }}"
+  - name: backport patches to 8.7 branch
+    conditions:
+      - merged
+      - label=backport-v8.8.0
```

Comment:
The changes to `mergify.yml` are incorrect - just leaving this review for reference